docs/sources/tutorials/alerting-get-started/index.md

```diff
@@ -48,27 +62,57 @@ To demonstrate the observation of data using the Grafana stack, download the fil
 
 1. Clone the [tutorial environment repository](https://www.github.com/grafana/tutorial-environment).
 
+   <!-- INTERACTIVE exec START -->
    ```
    git clone https://github.com/grafana/tutorial-environment.git
    ```
+   <!-- INTERACTIVE exec END -->
 
 1. Change to the directory where you cloned the repository:
 
+   <!-- INTERACTIVE exec START -->
    ```
    cd tutorial-environment
    ```
+   <!-- INTERACTIVE exec END -->
 
 1. Run the Grafana stack:
 
+   <!-- INTERACTIVE ignore START -->
    ```
    docker compose up -d
    ```
+   <!-- INTERACTIVE ignore END -->
+
+   {{< docs/ignore >}}
+
+   <!-- INTERACTIVE exec START -->
+    ```bash 
+    docker-compose up -d 
+    ```
+   <!-- INTERACTIVE exec END -->
+
+   {{< /docs/ignore >}}
 
    The first time you run `docker compose up -d`, Docker downloads all the necessary resources for the tutorial. This might take a few minutes, depending on your internet connection.
 
-   {{< admonition type="note" >}}
-   If you already have Grafana, Loki, or Prometheus running on your system, you might see errors, because the Docker image is trying to use ports that your local installations are already using. If this is the case, stop the services, then run the command again.
-   {{< /admonition >}}
+   
+   **NOTE:** If you already have Grafana, Loki, or Prometheus running on your system, you might see errors, because the Docker image is trying to use ports that your local installations are already using. If this is the case, stop the services, then run the command again.
+
+<!-- INTERACTIVE ignore START -->
+{{< admonition type="tip" >}}
+Alternatively, you can try out this example in our interactive learning environment: [Get started with Grafana Alerting](https://killercoda.com/grafana-labs/course/).
+
+It's a fully configured environment with all the dependencies already installed.
+
+![Interactive](https://raw.githubusercontent.com/grafana/killercoda/prod/assets/full-stack-ile.png)
```

Comment:
I was chatting with @JStickler about this. I think if possible we should upload the image via the Writers Tool Kit method:
https://admin.grafana.com/upload/media/docs/grafana/

```
![Interactive](/media/docs/grafana/full-stack-ile.png)
```

Reply:
no problemo, i will upload it

Reply:
nevermind, i see it's already there. I will just change the link then